src/abft-messages-proposals.md

```diff
@@ -1,30 +1,46 @@
+$$
+\newcommand \pk {\mathrm{pk}}
+\newcommand \fv {\text{first}}
+\newcommand \lv {\text{last}}
+\newcommand \Sign {\mathrm{Sign}}
+\newcommand \ValidEntry {\mathrm{ValidEntry}}
+\newcommand \Hash {\mathrm{Hash}}
+\newcommand \Digest {\mathrm{Digest}}
+\newcommand \Encoding {\mathrm{Encoding}}
+\newcommand \Record {\mathrm{Record}}
+\newcommand \Verify {\mathrm{Verify}}
+\newcommand \Proposal {\mathrm{Proposal}}
+$$
+
 # Proposals
 
-Let \\(e = (o, Q)\\) be an entry and \\(y\\) be the output of a \\(Sign\\) procedure.
+Let \\( e = (o, Q) \\) be an entry and \\( y \\) be the output of a \\( \Sign \\)
+procedure.
 
-The pair \\((e, y)\\) is a _proposal_ or a _proposal payload_.
+The pair \\( (e, y) \\) is a _proposal_ or a _proposal payload_.
 
 Moreover, let
 
-- \\(L\\) be a ledger where \\(|L| \geq \delta_b\\),
+- \\( L \\) be a ledger where \\( |L| \geq \delta_b \\),
 
-- \\(v = (I, p, h, x)\\) be some proposal-value.
+- \\( v = (I, p, h, x) \\) be some proposal-value.
 
-We say that this proposal is _a valid proposal matching \\(v\\) with respect to
-\\(L\\)_ (or simply that this proposal _matches \\(v\\)_ if \\(L\\) is unambiguous)
+We say that this proposal is _a valid proposal matching \\( v \\) with respect to
+\\( L \\)_ (or simply that this proposal _matches \\( v \\)_ if \\( L \\) is unambiguous)
 if the following conditions are true:
 
-- \\(ValidEntry(L, e) = 1\\),
+- \\( \ValidEntry(L, e) = 1 \\),
 
-- \\(h = Digest(e)\\),
+- \\( h = \Digest(e) \\),
 
-- \\(x = Hash(Encoding(e))\\),
+- \\( x = \Hash(\Encoding(e)) \\),
 
-- The seed \\(Q\\) and seed proof are valid as specified in the following section,
+- The seed \\( Q \\) and seed proof are valid as specified in the following section,
```

Comment:
@cce @jannotti the seed symbol has been renamed from `s` (original spec) to `Q`, since it is the symbol usually used for the seed.